pkg/mermaid/mermaidintegrationdiagram.go

```diff
@@ -0,0 +1,88 @@
+package mermaid
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/anz-bank/sysl/pkg/sysl"
+)
+
+type integrationpair struct {
+	firstapp, secondapp string
+}
+
+func GenerateMermaidIntegrationDiagram(m *sysl.Module, appname string) (string, error) {
+	var ip []integrationpair
+	return generateMermaidIntegrationDiagramHelper(m, appname, &ip, true)
+}
+
+func generateMermaidIntegrationDiagramHelper(m *sysl.Module, appname string,
+	integrationpairs *[]integrationpair, thestart bool) (string, error) {
```

Comment:
I'd suggest using camelCase for variable names in Go just to stick with convention https://golang.org/doc/effective_go.html#mixed-caps

Reply:
Yep okay, was confused on what the convention was so didn't pay much attention to it. Will make changes now.

Reply:
Addressed!